geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -644,9 +644,9 @@ def _fill_internal_wms(self, l, layer, wms, wms_layers, errors):
             resolutions = self._get_layer_resolution_hint(wms_layer_obj)
             if resolutions[0] <= resolutions[1]:
                 if "minResolutionHint" not in l:
-                    l["minResolutionHint"] = float("{0:0.2f}".format(resolutions[0]))
+                    l["minResolutionHint"] = float("{:0.2f}".format(resolutions[0]))
```

Comment:
What do you want to achieve, here? I don't understand the back and forth with string...

Reply:
Non functional change, just removing unneeded position